### ISSUE TITLE
Comment support

### ISSUE THREAD

**Issue by arauchberger** [boothen/Json-Eclipse-Plugin#31]
Hi,

i know that json is not intended to support comments, but parser like jackson do support this.
sometimes it is really very handy to have some comments in json-files.

is there any chance of getting comment-support (there is already color-support for comments ;-) for the plugin?

thanks in advance

![Screenshot 2021-04-15 at 12 47 54](https://user-images.githubusercontent.com/31243977/114857699-dcf5c580-9de8-11eb-9ddc-c85adcf12371.png)

**Comment by msteiger** [comment]
Hello, what exactly do you expect from such a support?

**Comment by arauchberger** [comment]
Hi Martin,

i would expect to get ride of the error-marker in eclipse for these json-comments.

![Screenshot 2021-04-16 at 08 04 15](https://user-images.githubusercontent.com/31243977/114978976-5ee9fb00-9e8a-11eb-9467-745433671ec1.png)

**Comment by msteiger** [comment]
Makes sense, I guess. Haven't touched it in quite a while, maybe you can make a PR yourself and I review it?
